aioesphomeapi/api.proto

```diff
@@ -846,7 +846,6 @@ message ListEntitiesClimateResponse {
   bool supports_target_humidity = 23;
   float visual_min_humidity = 24;
   float visual_max_humidity = 25;
-  bool supports_aux_heat = 26;
```

Comment:
Let's call this well planned that it ended up being the highest value

Reply:
Hehe, sure, all planned 😎